### ISSUE TITLE
GenIso for Scala 3

### ISSUE THREAD

**Issue by julien-truffaut** [optics-dev/Monocle#1042]
Delegate `GenIso[A](_.foo)` in Scala 3 implementation to Focus[A](_.foo)

Depends on #1032

see for GenLens https://github.com/optics-dev/Monocle/blob/master/macro/src/main/scala-3.x/monocle.macros/GenLens.scala

**Comment by kenbot** [comment]
This is unblocked now

**Comment by julien-truffaut** [comment]
Sorry I was wrong, the API for `GenIso` in Scala 2 is `GenIso[IntWrapper, Int]` and not `GenIso[IntWrapper](_.value)` where
```scala
case class IntWrapper(value: Int)
```

I am tempted to close this ticket as `wontfix`. I don't believe `GenIso[A, B]` was used very much and we can point people to `Focus` instead.

**Comment by kenbot** [comment]
> and we can point people to Focus instead.
Is that the action for this ticket then, or perhaps there is nothing to do?

**Comment by julien-truffaut** [comment]
Yeah I think so. Closing it.
